### PR TITLE
Late review on drop_connect docs

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Address late review comments on PR #29947.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104319
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jbouwh** [comment]
@pfrazer If you have time, please have a look.

File: source/_integrations/drop_connect.markdown [hunk 1]
```diff
@@ -60,7 +49,19 @@ To enable the DROP integration, use the DROP Connect app to connect your DROP Hu
...
 Assistant.
-Go to **Settings > Devices & Services** in the Home Assistant UI to add your DROP devices.
 
+{% details "Manual configuration steps" %}
+
+- Browse to your Home Assistant instance.
+- Go to **{% my integrations title="Settings > Devices & Services" %}**.
+- Set up the new discovered devices.
+
+{% enddetails %}
```
Comment (frenck) [inline source/_integrations/drop_connect.markdown:61]: This needs to include our config_flow.md template file

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/drop_connect.markdown (hunk 1 above)
Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:61]: This integration is not set up by adding a config entry manually. Devices need to be discovered, and after that they can be set up. By adding the `config_flow.md` template the wrong instructions are shown.

**Comment by pfrazer** [comment]
> @pfrazer If you have time, please have a look.

@jbouwh  I'm on vacation until the 1st with amazingly bad internet, so my involvement for the time being may be limited to comments like this...

I am fine with the changes that you made.  The only detail I see that needs attention is the note at the end:

> The **broker broker** address and port used for the DROP Hub must...

Change that to 'MQTT broker' or similar and it should be fine.

File: source/_integrations/drop_connect.markdown [hunk 2]
```diff
@@ -60,7 +49,19 @@ To enable the DROP integration, use the DROP Connect app to connect your DROP Hu
...
+
+<div class="note">
+
+The broker broker address and port used for the DROP Hub must be the same as configured for the Home Assistant MQTT integration.
```
Comment (jbouwh) [inline source/_integrations/drop_connect.markdown]:
```suggestion
The MQTT broker address and port used for the DROP Hub must be the same as configured for the Home Assistant MQTT integration.
```

File: source/_integrations/drop_connect.markdown (hunk 1 above)
Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:61]: I ended using part of the template code.

Comment (jbouwh) [reply source/_integrations/drop_connect.markdown:61]: @frenck may be you can have another look as using `config_flow.md` is not appropriate here.

**Review by frenck — APPROVED** [review]
Thanks, @jbouwh 👍 

../Frenck

**Pull request merged**